ambry-commons/src/main/java/com/github/ambry/clustermap/HelixVcrUtil.java

```diff
@@ -43,10 +41,34 @@
  */
 public class HelixVcrUtil {
   private static final Logger logger = LoggerFactory.getLogger(HelixVcrUtil.class);
-  public static final List<String> ignoreResourceKeyWords = Arrays.asList("aggregation", "trigger", "stats");
   static final String SEPARATOR = "/";
   private static final ZNRecordSerializer ZN_RECORD_SERIALIZER = new ZNRecordSerializer();
   private static final String DELAYED_REBALANCER_CLASS_NAME = "DelayedAutoRebalancer";
+  private static final Set<String> smallIntegers = new HashSet<>();
+
+  static {
+    for (int i = 0; i < 128; i++) {
+      smallIntegers.add(String.valueOf(i));
+    }
+  }
+
+  /**
+   * Return {@code true} is the given resource name is a resource that contains partition information.
+   * @param resourceName The resource name.
+   * @return {@code true} when the given resource contains partition inforrmation
+   */
+  public static boolean isPartitionResourceName(String resourceName) {
```

Comment:
I know this is out of scope for this commit, but should we do something to ensure that any new/old non-partition resources created by Ambry in helix are not integer?